### PR TITLE
--backupdb changes

### PR DESCRIPTION
**Pull request by ThomasTJdev**
Allow for backup db without gpg signing.

Create folder `backup` to store backups in.

Use `info()` on messages.

### CODE REVIEW

**Comment by ThomasTJdev** [comment]
Not tested on postgres, only SQLite.

Question: Should the host be from config, when it is postgres? 
```
backupDb*(dbname: string,
    filename = fileBackup & replace($now(), ":", "_") & ".sql",
    host = "localhost"
    ...``

File: nimwcpkg/resources/administration/createdb.nim [hunk 1]
```diff
@@ -198,37 +198,51 @@ proc generateDB*(db: DbConn) =
 
 
 proc backupDb*(dbname: string,
-    filename = fileBackup & replace($now(), ":", "_") & ".sql",
+    filename = "backup" / fileBackup & replace($now(), ":", "_") & ".sql",
     host = "localhost", port = Port(5432), username = getEnv("USER", "root"),
     dataOnly = false, inserts = false, checksum = true, sign = true, targz = true,
-    ): tuple[output: TaintedString, exitCode: int] =
+    gpg = false): tuple[output: TaintedString, exitCode: int] =
```
Comment (juancarlospaco) [inline nimwcpkg/resources/administration/createdb.nim]: `sign = false` Already does this on the code, is exactly the same, the feature is duplicated. :hushed:

File: nimwc.nim [hunk 2]
```diff
@@ -391,7 +392,8 @@ when isMainModule:
         if not styleExist:
           echo("\nStandard data: Please specify data style:\n  1) bulma\n  2) bootstrap\n  3) clean\n!! Data not inserted !!")
       of "vacuumdb": echo vacuumDb(db)
-      of "backupdb": echo backupDb(cfg.getSectionValue("Database", when defined(postgres): "name" else: "host"))
+      of "backupdb-gpg": echo backupDb(cfg.getSectionValue("Database", when defined(postgres): "name" else: "host"), gpg=true)
+      of "backupdb": echo backupDb(cfg.getSectionValue("Database", when defined(postgres): "name" else: "host"), gpg=false)
```
Comment (juancarlospaco) [inline nimwc.nim]:
```suggestion
      of "backupdb": echo backupDb(cfg.getSectionValue("Database", when defined(postgres): "name" else: "host"), sign=false)
```

File: nimwcpkg/resources/administration/createdb.nim [hunk 3]
```diff
@@ -198,37 +198,51 @@ proc generateDB*(db: DbConn) =
...
-  else:  # SQLite .dump is Not working, Docs says it should.
+  else:  # TODO: SQLite .dump is Not working, Docs says it should.
     var cmd = cmdBackup.format(dbname, filename)
-  when not defined(release): echo cmd
+
+  when not defined(release): info("Database backup: " & cmd)
   result = execCmdEx(cmd)
-  if checksum and result.exitCode == 0 and findExe"sha512sum".len > 0:
+
+  if gpg and checksum and result.exitCode == 0 and findExe("sha512sum").len > 0:
```
Comment (juancarlospaco) [inline nimwcpkg/resources/administration/createdb.nim]:
```suggestion
  if checksum and result.exitCode == 0 and findExe("sha512sum").len > 0:
```

File: nimwcpkg/resources/administration/createdb.nim (hunk 1 above)
Comment (juancarlospaco) [inline nimwcpkg/resources/administration/createdb.nim]:
```suggestion
    ): tuple[output: TaintedString, exitCode: int] =
```

**Comment by juancarlospaco** [comment]
`sign = false` Already does this on the code, is exactly the same, the feature is duplicated. :hushed:

File: nimwcpkg/resources/administration/createdb.nim (hunk 1 above)
Comment (ThomasTJdev) [reply nimwcpkg/resources/administration/createdb.nim]: You right, I was to fast. My mistake, sorry.

File: nimwc.nim (hunk 2 above)
Comment (ThomasTJdev) [reply nimwc.nim]: Fixed 92dae8a5bb3a5437c055238caa62e4ae14216750

Using `, checksum=false, sign=false, targz=false` to clarify all options.

File: nimwcpkg/resources/administration/createdb.nim (hunk 3 above)
Comment (ThomasTJdev) [reply nimwcpkg/resources/administration/createdb.nim]: Fixed 92dae8a

File: nimwcpkg/resources/administration/createdb.nim (hunk 1 above)
Comment (ThomasTJdev) [reply nimwcpkg/resources/administration/createdb.nim]: Fixed 92dae8a

**Pull request merged**